java16/src/main/java/io/papermc/paperclip/Paperclip.java

```diff
@@ -268,8 +269,10 @@ private static void mavenInstall(Path paperJar) {
                 System.err.println("No Paper pom file could be found.");
                 System.exit(1);
             }
-
-            if (new ProcessBuilder(mavenCommand, "install:install-file", "-Dfile=" + paperJar, "-DpomFile=" + pomPath).start().waitFor() != 0) {
+            Process installProcess = new ProcessBuilder(mavenCommand, "install:install-file", "-Dfile=" + paperJar, "-DpomFile=" + pomPath)
```

Comment:
Maybe add back the empty line here to keep the separation between installation and pom extraction visible in the code (I am being super nit-picky tho).
While add it, make the `installProcess` variable `final` as is custom with all other variables you find in the method.